src/keypoints/keypoints.h

```diff
@@ -35,6 +35,12 @@ class KeyPoints {
     this->size = kp.size;
     this->kpoints = kp.kpoints;
   }
+
```

Comment:
I added tha allocate function because if you initialize a KeyPoints with the default constructor it will not init the `kpoints`.